build.rs

```diff
@@ -205,10 +209,50 @@ fn ignore(testsuite: &str, testname: &str, strategy: &str) -> bool {
                 return true;
             }
 
+            // FIXME(#1569) stack protection isn't implemented yet and these
+            // tests segfault.
+            ("spec_testsuite", "skip_stack_guard_page")
+            | ("spec_testsuite", "stack")
+            | ("misc_testsuite", "stack_overflow")
+                if target.contains("aarch64") =>
+            {
+                return true
+            }
+
             _ => {}
         },
         _ => panic!("unrecognized strategy"),
     }
 
     false
 }
+
+fn should_panic(testsuite: &str, testname: &str) -> bool {
```

Comment:
Little clarity nit, but maybe add a comment here to distinguish from the "test that this correctly panics" unit-test idiom. Something like ```// Determine whether to add a should_panic attribute. These tests currently panic because of unfinished backend implementation work; we will remove them from this list as we finish the implementation.```